2/camille-baldock-captions.srt

```diff
@@ -712,8 +718,8 @@ like it to say 'if it's a 5 of Diamond' it takes
 
 164
 00:11:28,480 --> 00:11:32,600
-card.move 5 and a
-diamond symbol type thing.
+Card.new 5 and a
+Diamond symbol type thing.
```

Comment:
I like the way you've done it. It makes for a nice balance between capturing the spoken word and the written code.